src/server/pkg/serviceenv/service_env.go

```diff
@@ -107,7 +107,11 @@ func (env *ServiceEnv) initPachClient() error {
 	// Initialize pach client
 	return backoff.Retry(func() error {
 		var err error
-		env.pachClient, err = client.NewFromAddress(env.pachAddress)
+		var opts []client.Option
```

Comment:
This is unnecessary for the time being, and will just be removed after 1.12. Although I love the functional option pattern, I don't think we should bother setting it up for the V2 config since the configuration will be removed after 1.12. The current approach for enabling V2 is by propagating the `STORAGE_V2` environment variable to each container and having the client check for this at creation time. I think this pattern will accomplish what we need in the short term, and will be easier to remove when V2 is no longer a configuration.

Reply:
We need to be able to pass this programmatically, and not rely on a global variable, if we want the tests to run concurrently. That is not an issue now.